### PR TITLE
README: fix typo

### PR DESCRIPTION
**Pull request by l2dy**
### What problem does this PR solve? 
Fix number of CRD objects.

### CODE REVIEW

**Comment by cwen0** [comment]
@l2dy Thanks for your contribution! Please sign CLA, thanks!

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ custom definitions for chaos experiments and automatic orchestration. There are
 ![Chaos Operator](./static/chaos-mesh.svg)
 
 Chaos Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to define chaos objects. 
-The current implementation supports three types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, and TimeChaos, 
+The current implementation supports four types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, and TimeChaos,
```
Comment (cwen0) [inline README.md:38]: The current implementation supports six types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, TimeChaos, StresssChaos, and KernelChaos.

Comment (cwen0) [inline README.md:38]: The current implementation supports six types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, TimeChaos, StresssChaos, and KernelChaos.

Comment (l2dy) [reply README.md:38]: In this case, I think I should close the PR and leave it to you ;)

Comment (cwen0) [reply README.md:38]: This is indeed a typo, You can reopen this pr and fix it in this pr, I will pull a new pr to add the new chaos. Thanks!

Comment (l2dy) [reply README.md:38]: I'd rather not sign the CLA just for a change to be overridden in the next commit. Thanks for your help anyway.

Comment (zhouqiang-cl) [reply README.md:38]: thank you @l2dy

**Pull request closed without merging**